jupyterlab_latex/__init__.py

```diff
@@ -117,13 +119,19 @@ def build_tex_cmd_sequence(self, tex_base_name, run_bibtex=False):
             escape_flag = '-no-shell-escape'
         elif c.shell_escape == 'restricted':
             escape_flag = '-shell-restricted'
-            
+
+        # Get the synctex query parameter, defaulting to
+        # 1 if it is not set or is invalid.
+        synctex = self.get_query_argument('synctex', default='1')
```

Comment:
Interesting, I didn't know you could get query arguments in this way, I thought it required changing the actual signature of the get method & parsed. This is much cleaner.

Why do we want to do this in the `initialize` method? I think we're going to want to refactor this when we do the splitting up into separate files. 

I want to point out that in the future I'd prefer to preëmptively split into files rather than to do so only once it's gotten too bad, because now the splitting seems like a much larger and more complicated task than it would have had we done it earlier.

Reply:
This is not in the `initialize` method?

In the future, I have no problem with earlier splitting.

Reply:
Yeah, I like the query-string customization of the behavior (rather than a more complex endpoint, or a `post` body).

Reply:
oh oops… should have expanded the code. on Github… that makes sense. and yes to all of that regarding complexity & get_query_string